mlflow/pipelines/steps/train.py

```diff
@@ -159,6 +159,7 @@ def _validate_and_apply_step_config(self):
                 error_code=INVALID_PARAMETER_VALUE,
             )
 
+        self.output_probabilities = self.step_config.get("output_probabilities", False)
```

Comment:
Can we change this `output_probabilities` to `predict_proba` to reduce the cognitive (over)load a bit?